pkg/sensors/tracing/kprobe_sigkill_test.go

```diff
@@ -113,6 +114,41 @@ func TestKprobeSigkill(t *testing.T) {
 	testSigkill(t, makeSpecFile, checker)
 }
 
+func TestKprobeSigkillExecveMap1(t *testing.T) {
+	if !kernels.MinKernelVersion("5.3.0") {
+		t.Skip("sigkill requires at least 5.3.0 version")
+	}
+
+	// makeSpecFile creates a new spec file bsed on the template, and the provided arguments
```

Comment:
nit: we don't really care but since I saw it, little typo here `bsed` to `based`